docs/source/examples/performance_modeling_output.txt

```diff
@@ -0,0 +1,55 @@
+                                   \vec{Re} : [ 7.56e+05  7.56e+05  7.56e+05  7.56e+05 ]        Reynold's number 
+
+Constants
+---------
+                         Aircraft, Fuselage |                                                                          
+                                          W : 100                                         [lbf]       weight           
+                                                                                                                       
+                             Aircraft, Wing |                                                                          
+                                          A : 27                                                      aspect ratio     
+                                          S : 190                                         [ft**2]     surface area     
+                                       \rho : 1                                           [lbf/ft**2] areal density    
+                                                                                                                       
+Mission, FlightSegment, AircraftP, WingAero |                                                                          
+                                    \vec{e} : [ 0.9       0.9       0.9       0.9      ]              Oswald efficiency
+                                                                                                                       
+        Mission, FlightSegment, FlightState |                                                                          
+                                    \vec{V} : [ 40        40        40        40       ]  [kt]        true airspeed    
+                                  \vec{\mu} : [ 1.63e-05  1.63e-05  1.63e-05  1.63e-05 ]  [N*s/m**2]  dynamic viscosity
+                                 \vec{\rho} : [ 0.74      0.74      0.74      0.74     ]  [kg/m**3]   air density      
+
+Sensitivities
+-------------
+Mission, FlightSegment, FlightState |                                                         
+                            \vec{V} : [ 0.0997    0.1       0.101     0.102    ] true airspeed
+                                                                                              
+                     Aircraft, Wing |                                                         
+                                  S : 0.6831                                     surface area 
+                               \rho : 0.4816                                     areal density
+                                  A : -0.3056                                    aspect ratio 
+                                                                                              
+                 Aircraft, Fuselage |                                                         
+                                  W : 0.2535                                     weight       
+
```

Comment:
We're indenting right now just to align the `|` with the `:`, but we could instead do

```
Aircraft, Fuselage
          W : 0.2535
```

or 

```
            | Aircraft, Fuselage
          W : 0.2535
```